runtime/queries/ocaml/highlights.scm

```diff
@@ -1,14 +1,17 @@
 ; Modules
 ;--------
 
-[(module_name) (module_type_name)] @namespace
+[(module_name) (module_type_name)] @module
```

Comment:
Ah actually I missed this one. This should be reset as well (to match https://docs.helix-editor.com/master/themes.html#syntax-highlighting scopes)